File: datasets/java/valid/Valid0106.java
public class Valid0106 {
    private int value;
    
    public Valid0106(int value) {
        this.value = value;
    }
    
    public int getValue() {
        return value;
    }
    
    public static void main(String[] args) {
        Valid0106 obj = new Valid0106(42);
        System.out.println("Value: " + obj.getValue());
    }
}

File: datasets/python/valid/valid_0106.py
class Processor:
    def __init__(self, res):
        self._res = res

    def get_res(self):
        return self._res

    def set_res(self, x):
        self._res = x

obj = Processor(33)
print(obj.get_res())
obj.set_res(25)
print(obj.get_res())
